core/modules/filters/getIndex.js

```diff
@@ -0,0 +1,32 @@
+/*\
+title: $:/core/modules/filters/getIndex.js
```

Comment:
It's inconsistent to use "getIndex"; all the other two word filters just concatenate lower case words (eg, "removeprefix", "moduletypes"). The filename needs changing too.